weather.py

```diff
@@ -68,7 +84,22 @@ def find_min(weather_data):
     Returns:
         The minium value and it's position in the list.
     """
-    pass
+    if not weather_data:
+        return ()
+    else:
+        min_value = float(9999999)
+        min_index = None
+        index = 0
+
+        for this_element in weather_data:
+            this_element = float(this_element)
+            if this_element <= min_value:
+                min_value = this_element
+                min_index = index
+
+            index += 1
+
+        return (float(min_value), min_index)
```

Comment:
Since `this_element` has been turned into a `float`, `min_value` will also always be a `float`, so no need to re-cast the type here.